english/exercise/exercise.md

```diff
@@ -0,0 +1 @@
+If you [eat too much mashmallows](https://github.com/udacity/create-your-own-adventure/blob/master/english/eating-walls/eating-marshmallows.md), you will gain weight. Therefore, be sure to go for a run and do some exercise, lose some calories and then [eat](https://github.com/udacity/create-your-own-adventure/blob/master/english/eating-walls/eating-marshmallows.md) again.
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.